pages/installing/gettingStarted-Humming.md

```diff
@@ -0,0 +1,217 @@
+# Getting Started With the Humming Board
+
+![Humming Board](/img/hummingBoard.png)
+
+## What You'll Need
+
+* A [Humming Board i2][humming-link] from [Solid Run][solid-run]. Currently we only support the `i2` device type.
+
+* A 4GB or larger SD card.
+  The [Humming Board i2][humming-link] uses a Micro SD card. The [speed class][speed_class] of
```

Comment:
You use `Micro SD` here, but in a bunch of other places you use `microSD`, it's probably worth being consistent